oasislmf/computation/run/model.py

```diff
@@ -38,6 +40,19 @@ class RunModel(ComputationStep):
     ]
 
 
+    def update_kwargs(self):
+        oed_filenames = OrderedDict([f for f in SOURCE_FILENAMES.items() if f[0].startswith('oed_')])
+        updated_inputs = dict()
+
+        for f in oed_filenames:
+            if self.kwargs[f]:
+                updated_inputs[f] = os.path.join(
+                    self.kwargs['oasis_files_dir'],
+                    oed_filenames[f]
```

Comment:
I think you should use the function store_exposure_fp(fp, exposure_type) (in utils.default)
to keep the extension.